backend/src/routes/api/k8s/pass-through.ts

```diff
@@ -39,7 +39,7 @@ export const passThrough = <T extends K8sResourceCommon>(
           };
         } else {
           // Likely not JSON, print the error and return the content to the client
-          fastify.log.error(`Parsing response error: ${e}, ${data}`);
+          fastify.log.error(e, data, 'Parsing response error');
```

Comment:
2nd param is the message while any subsequent params are used to format the message string.
```suggestion
          fastify.log.error(e, `Parsing response error: ${data}`);
```